content/projects/open-alias/index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "OpenAlias"
+date: 2018-07-09T11:41:44+12:00
+draft: false
+author: "Alex Vasquez"
```

Comment:
correct spelling would be Vazquez

Reply:
@aguycalled fixed and also stripped out the informational text. was supposed to be used as guide for content team. will add remaining copy when they have it finalised.